### ISSUE TITLE
Don't say what install won't do?

### ISSUE THREAD

**Issue by hadley** [r-lib/pak#97]
```
→ Will install 5 packages:
  KernSmooth, Matrix, nnet, rpart, survival
  
→ Will update 2 packages:
  recipes, tidyr
  
→ Will not update 32 packages.

→ Will download 6 CRAN packages (12.2 MB).
→ Will download 1 packages with unknown size.
```

I think it would be fine to omit what pak is not doing (and close up the whitespace a bit):

```
→ Will install 5 packages:
  KernSmooth, Matrix, nnet, rpart, survival  
→ Will update 2 packages:
  recipes, tidyr

→ Will download 6 CRAN packages (12.2 MB).
→ Will download 1 packages with unknown size.
```

I wonder if there's some way to indicate that the installing/updating and downloading are referring to the same packages.

**Comment by gaborcsardi** [comment]
> I think it would be fine to omit what pak is not doing (and close up the whitespace a bit):

IDK, that's still useful information, i.e. that your `n` dependencies will not be updated.

**Comment by hadley** [comment]
In that case, I'd reword a bit. Some ideas:

```
→ Will not update 32 dependencies.
→ Will leave 32 dependencies as is.
→ 32 dependencies remain unchanged.
```

**Comment by gaborcsardi** [comment]
Well, except that it is not always dependencies. E.g. you `pkg_install(pkg)`, and `pkg` is already installed, but one of its dependencies is missing. Then only that dependency is installed and `pkg` is left unchanged.

**Comment by hadley** [comment]
Oh hmmmm, maybe "will leave 32 related packages as is"?

Also would be useful to see this when no changes are made:

```R
> pkg_install("stringr")
ℹ Checking for package metadata updates
✔ Downloaded metadata files, 456.36 kB in 1 files.                              
✔ Updating metadata database
✔ Using cached package metadata                                                 
✔ 1 + 3 pkgs | kept 2, updated 0, new 0 | downloaded 0 (0 B) [7.4s]             
 12  deps for  1/1 
```

**Comment by hadley** [comment]
I'm going to relax my suggestion a bit to just remove the whitespace and add "new" and "existing"

```
→ Will install 5 new packages:
  KernSmooth, Matrix, nnet, rpart, survival  
→ Will update 2 existing packages:
  recipes, tidyr 
→ Will not update 32 existing packages.

→ Will download 6 CRAN packages (12.2 MB).
→ Will download 1 packages with unknown size.
```

**Comment by gaborcsardi** [comment]
These messages were already removed in https://github.com/r-lib/pak/commit/617425ba3c2d43eedb570b880a1b9701fdeb3657#diff-ce467f83a1655227364981632133bb17
